Assets/HoloToolkit-Examples/UX/Scripts/InteractiveSet.cs

```diff
@@ -22,7 +22,7 @@ public class InteractiveSet : MonoBehaviour
         public SelectionType Type = SelectionType.single;
 
         [Tooltip("Interactives that will be managed by this controller")]
-        public InteractiveToggle[] Interactives;
+        public List<InteractiveToggle> Interactives;
```

Comment:
What's the advantage of changing this to a list?

Reply:
It's easier to remove items in a list, see the "RemoveInteractive"-method in the same file. If the data changes I don't delete all Interactives and their prefabs, only the delta and update all items afterwards (recycling objects).